Road/endpoints/endpoints.py

```diff
@@ -1,6 +1,7 @@
 from http import HTTPStatus
 from fastapi import Request
 from Road import app, logger
+from Road.source.utils import read_n_from_file
```

Comment:
You should call the function and save the returned value to pass it to processed functions.